gui/elements/_viewer.py

```diff
@@ -74,6 +64,45 @@ def camera(self):
         """
         return self._view.camera
 
+    @property
+    def status(self):
+        """string: Status string
+        """
+        return self._status
+
+    @status.setter
+    def status(self, status):
+        if status == self.status:
+            return
+        self._status = status
+        self.events.status(text=self._status)
+
+    @property
+    def help(self):
+        """string: Help string
+        """
+        return self._help
+
+    @help.setter
+    def help(self, help):
+        if help == self.help:
+            return
+        self._help = help
+        self.events.help(text=self._help)
+
+    @property
+    def annotation(self):
+        """bool: Annotation mode
+        """
+        return self._annotation
+
+    @annotation.setter
+    def annotation(self, annotation):
+        # if annotation == self.annotation:
```

Comment:
Why is this commented out?

Reply:
Right now I need the event to emit when the annotation mode is set but not changed. This is because of how some of the cursor updates are done. It seems though that this is really a bit of an anti-pattern though, and I can try and fix!!

Reply:
I have fixed this now, thanks for flagging it